src/pages/office/components/termination-code/termination-code.scss

```diff
@@ -0,0 +1,8 @@
+.mes-select {
+  .alert-wrapper div.alert-radio-group {
+    max-height: 800px;
```

Comment:
Again, this would target all `mes-select` boxes, so see comment below about maybe adding another class to the `ion-select` so you can target just this.
You could then use something like `.mes-select.termination-code-select { ... }`

If the change is for all boxes, the styling should go in `_alert.scss`

Reply:
Good spot, didn't realise `mes-select` was being used across the app